src/app/ApplicationTemplate.Mobile/ApplicationTemplate.Mobile.csproj

```diff
@@ -0,0 +1,154 @@
+				<!-- See https://github.com/xamarin/xamarin-macios/issues/14812 for more details. -->
+				<MtouchExtraArgs>$(MtouchExtraArgs) --marshal-objectivec-exceptions:disable</MtouchExtraArgs>
+			</PropertyGroup>
+			<Choose>
+				<When Condition="'$(Configuration)'=='Debug'">
+					<PropertyGroup>
+						<MtouchExtraArgs>--optimize=-remove-dynamic-registrar --xml=./iOS/LinkerExclusions.xml --linkskip=$(AssemblyName) --setenv=MONO_LOG_LEVEL=debug --setenv=MONO_LOG_MASK=gc --setenv=MONO_GC_PARAMS=soft-heap-limit=512m,nursery-size=64m,evacuation-threshold=66,major=marksweep,concurrent-sweep</MtouchExtraArgs>
+					</PropertyGroup>
+					<PropertyGroup Condition="'$(Platform)'=='iPhone'">
+						<CodesignKey>iPhone Developer</CodesignKey>
+						<CodesignEntitlements>./iOS/Entitlements.plist</CodesignEntitlements>
+						<MtouchDebug>True</MtouchDebug>
+						<Optimize>False</Optimize>
+					</PropertyGroup>
+					<PropertyGroup Condition="'$(Platform)'=='iPhoneSimulator'">
+						<CodesignEntitlements>./iOS/Entitlements.plist</CodesignEntitlements>
+						<MtouchDebug>True</MtouchDebug>
+						<Optimize>False</Optimize>
+					</PropertyGroup>
+				</When>
+				<When Condition="'$(Configuration)'=='Release'">
+					<PropertyGroup>
+						<BuildIpa>True</BuildIpa>
+						<CodesignEntitlements>./iOS/Entitlements.plist</CodesignEntitlements>
+						<CodesignKey>iPhone Distribution</CodesignKey>
+						<RuntimeIdentifier>ios-arm64</RuntimeIdentifier>
+					</PropertyGroup>
+					<Choose>
+						<When Condition="'$(IsLightBuild)'=='True'">
+							<!-- Build Optimization for LightBuild (Faster Builds in Pull Requests). -->
+							<PropertyGroup>
+								<Optimize>False</Optimize>
+								<!-- Prevents PNG Optimization. -->
+								<OptimizePNGs>False</OptimizePNGs>
+								<!-- Disables LLVM. -->
+								<MtouchUseLlvm>False</MtouchUseLlvm>
+								<MtouchUseThumb>False</MtouchUseThumb>
+								<!-- Disables Symbol Stripping. -->
+								<MtouchNoSymbolStrip>True</MtouchNoSymbolStrip>
+								<!-- Disables DSYM Creation. -->
+								<MtouchExtraArgs>$(MtouchExtraArgs) --dsym=false</MtouchExtraArgs>
+							</PropertyGroup>
+						</When>
+						<Otherwise>
+							<PropertyGroup>
+								<OptimizePNGs>True</OptimizePNGs>
+								<MtouchUseLlvm>True</MtouchUseLlvm>
+								<MtouchUseThumb>True</MtouchUseThumb>
+								<MtouchNoSymbolStrip>False</MtouchNoSymbolStrip>
+								<MtouchExtraArgs>--xml=./iOS/LinkerExclusions.xml --linkskip=$(AssemblyName) --setenv=MONO_GC_PARAMS=soft-heap-limit=512m,nursery-size=64m,evacuation-threshold=66,major=marksweep,concurrent-sweep</MtouchExtraArgs>
+								<Optimize>True</Optimize>
+							</PropertyGroup>
+						</Otherwise>
+					</Choose>
+				</When>
+			</Choose>
+			<ItemGroup>
+				<PackageReference Include="Uno.Extensions.Logging.OSLog" Version="1.4.0" />
+			</ItemGroup>
+			<ItemGroup>
+				<LinkDescription Include="iOS\LinkerExclusions.xml" />
```

Comment:
The linker for iOS is present but seems to be missing for Android is the csproj

Reply:
That's right, I forgot. Will have to do some research because it don't know if it's useful.

Reply:
It is useful for some packages we want the linker to ignore when there is not direct binding reference to it.